src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+import java.util.Optional;
+
+public class OperationStrategy {
+    private final Map<FruitTransaction.Operation, CountStrategy> countStrategyMap;
+
+    public OperationStrategy(Map<FruitTransaction.Operation, CountStrategy> countStrategyMap) {
+        this.countStrategyMap = countStrategyMap;
+    }
+
+    public CountStrategy getCountStrategyMap(FruitTransaction.Operation operation) {
```

Comment:
```suggestion
    public CountStrategy getCountStrategy(FruitTransaction.Operation operation) {
```
it dosen't return map?